pom.xml

```diff
@@ -26,9 +26,12 @@
     <folio-spring-base.version>7.1.2</folio-spring-base.version>
     <folio-query-tool-metadata.version>1.0.0</folio-query-tool-metadata.version>
     <mapstruct.version>1.5.2.Final</mapstruct.version>
-    <lib-fqm-query-processor.version>1.0.0</lib-fqm-query-processor.version>
+    <lib-fqm-query-processor.version>modfqmmgr-87-SNAPSHOT</lib-fqm-query-processor.version>
```

Comment:
This should be `1.1.0-SNAPSHOT`, but I wanted to test against the other branch in CI.  Will update once the `lib-fqm-query-processor` PR is merged.